apps/website/src/components/OutputTitle/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './styles.module.css';
```

Comment:
These should `makeStyles()` call, but Docosaurus doesn't support CSS-in-JS hooks for SSR setup: https://github.com/facebook/docusaurus/issues/3236